### PR TITLE
Version 1.26

### PR DESCRIPTION
**Pull request by nmanu1**
### Features
- Add support for the auth token, with a new global_config `token` field  (#976)
- Add support for the visitor, which can be set or changed at runtime (#970)
- Add support for generic thumbs up/down feedback buttons to all cards (#973, #1005)
- Handle page and template names with spaces (#988)

### Changes
- Update styling of navigation ‘More’ menu (#989)
- Update hover color of search bar buttons (#1000)
- Internal repo changes (automate WCAG testing (#987))

### Bugfixes
- Fix console error which would appear on google maps (#955)
- Fix extra query when loading in iframe integrations (#986)
- Fix bug that displayed links that didn’t exist (#990)
- Allow microphone access for voice search in iframe (#991)
- Fix issue with custom search icon size (#992)
- Fix bug with new prop comments not being added (#1004)
- Fix distance alignment on location cards (#1006)

### CODE REVIEW

File: test-site/config/global_config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "sdkVersion": "1.11", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```
Comment (yen-tt) [inline test-site/config/global_config.json:2]: make sure you update this to the latest sdk version!

File: global_config.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
-  "sdkVersion": "1.11", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```
Comment (yen-tt) [inline global_config.json:2]: make sure you update this to the latest sdk version!

Comment (nmanu1) [reply global_config.json:2]: yes, will do once the sdk is cut!

**Comment by cea2aj** [comment]
For "Fix console error which would appear on google maps", can we reference #955 rather than #981 which is a revert?

**Review by yen-tt — APPROVED** [review]

**Review by cea2aj — APPROVED** [review]

**Pull request merged**